README.md

```diff
@@ -1,17 +1,69 @@
 # PLEASE READ!
----
+
 
 This is a Selenium bot for buying a given sneaker from the Nike Snkrs website ON RELEASE DAY. 
    It will not work for sneakers that are past their release day (or later in the release day). 
    `Please note that this script was written with the US site in mind, so Nike sites for other countries will likely cause problems.`
    This is because the purchase page changes to where the buy button redirects to a seperate checkout page (as opposed to a direct buy popup during release).
 It is a commandline script written solely in python. `Please run with python 3.7.`
-There are 4 selenium drivers in the bin directory for both Chrome and Firefox on both Linux and MacOS. `I have not added the drivers for Windows.`
+There are 6 selenium drivers in the bin directory for both Chrome and Firefox on Linux, MacOS, and Winows. `The drivers for other operating systems need to be installed from the instructions below.`
 I have found the Firefox driver for MacOS works best.
 
 Ideally, some pieces (or all?) of this could be replaced with direct Nike API requests instead of Selenium. However, I've found that Nike APIs are not very straightforward. 
 
-Before running, make sure to install the requirements by running: `pip install -r requirements.txt`
```

Comment:
Prolly should keep this line in the README.

Reply:
Woop, don't know how that slipped by! Added back in